prog/dftb+/lib_solvation/cosmo.F90

```diff
@@ -397,7 +437,11 @@ function getRCutoff(this) result(cutoff)
     !> Resulting cutoff
     real(dp) :: cutoff
 
-    cutoff = 0.0_dp
+    if (allocated(this%sasaCont)) then
+      cutoff = max(cutoff, this%sasaCont%getRCutoff())
+    else
+      cutoff = 0.0_dp
```

Comment:
This should probably be before the if test, as the max branch can get an undefined value of cutoff, i.e.:
cutoff = 0.0_dp
if (allocated(allocated(this%sasaCont)) then
cutoff = max(cutoff, this%sasaCont%getRCutoff())
end if